README.md

```diff
@@ -1,6 +1,6 @@
 # pgAdmin 4 
 
-pgAdmin 4 is a rewrite of the popular pgAdmin3 management tool for the
+pgAdmin 4 is a rewrite of the popular pgAdmin 3 management tool for the
```

Comment:
The correct name was actually pgAdmin III.

Reply:
Thanks for the information.